.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
+}
```

Comment:
This will automatically format code based on the prettier standards whenever you save in VSCode. Seems easier than always having to manually run the script.